README.md

```diff
@@ -149,10 +149,10 @@ metadata:
   name: cluster-example
 spec:
   apiserverURL: "https://10.30.43.43:6443"
-  caData: ""
-  tokenData: ""
-  certData: ""
-  keyData: ""
+  caData:
```

Comment:
There is no need to remove this, `""` is still valid []byte

Reply:
but I got an error:
```
# pediaclusters.clusters.clusterpedia.io "cluster-x" was not valid:
# * spec.tokenData: Invalid value: "": spec.tokenData in body must be of type byte: ""
```

Reply:
Well, I thought this validation rule was the same as JSON of Golang

Reply:
LGTM